frontend/public/module/k8s/index.ts

```diff
@@ -79,8 +80,8 @@ export type Toleration = {
 };
 
 export type K8sResourceKind = {
-  apiVersion: string;
-  kind: string;
+  apiVersion?: string;
```

Comment:
to conform to the API: 
for example please see `"io.k8s.api.core.v1.Pod": {` in https://raw.githubusercontent.com/openshift/origin/master/api/swagger-spec/openshift-openapi-spec.json

Reply:
These are optional types mainly because when fetching a list of k8s resources (using `k8sList()`), the `apiVersion` and `kind` are often omitted from the individual list items.